bin/scotty.js

```diff
@@ -159,10 +164,19 @@ function cmd(console) {
     .then(() => beamUp(args, args.region, console))
 }
 
+
 function beamUp (args, region, console) {
-  return scotty(args.source, args.bucket, region, args.website, args.spa, args.update, args.force, args.quiet, console)
-    .then(endpoint => clipboardy.write(endpoint))
-    .then(() => process.exit(0))
+  const options = Object.assign({
+    logger: console
+  }, args, {region})
+  return scotty(options)
+    .then(endpoint => {
+      return new Promise(resolve => {
+        clipboardy.write(endpoint)
```

Comment:
you have `writeAsync` with `clipboardy`, so there is no need for a Promise wrap :)